Cauli/Florets/Mock/MockFloret.swift

```diff
@@ -67,6 +67,8 @@ public class MockFloret: InterceptingFloret {
 
     private let mode: Mode
 
+    public var description: String? = "Mock network requests for tests or to reproduce a bug. You can also use it to record your mock data."
```

Comment:
Please comment better descriptions for the `MockFloret`

Reply:
My proposal:

```swift

public var description: String? {
  if mode == .record {
    return "Enable to start recording network requests."
  } else {
    return "Enable to mock network requests. This floret can be configured to record requests as well."
  }
}
```